login-workflow/src/new-architecture/screens/VerifyCodeScreen/types.ts

```diff
@@ -0,0 +1,21 @@
+import { WorkflowCardProps } from '../../components/WorkflowCard/WorkflowCard.types';
```

Comment:
This is not related to this PR. @manojleaton We should have moved individual type definitions of WorkflowCard, WorkflowCardActions, WorkflowCardBody, WorkflowCardHeader, WorkflowCardInstructions to their respective files/folders and export them using index.ts.

@joebochill @daileytj